typebackend/models.py

```diff
@@ -24,3 +24,18 @@ class Paragraph(models.Model):
 
     def __str__(self):
         return self.taken_from
+
+class DashboardData(models.Model):
+    user=models.ForeignKey(settings.AUTH_USER_MODEL,on_delete=models.CASCADE,null=True)
```

Comment:
Can we make it OneToOne cause each user should be having one dashboard data object only.

Reply:
Why null=True?